specification/loadtestservice/LoadTestService.Management/routes.tsp

```diff
@@ -0,0 +1,104 @@
+import "./models.tsp";
+import "./customOperation.tsp";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.LoadTestService;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@@encodedName(LoadTests.createOrUpdate::parameters.resource,
+  "application/json",
+  "LoadTestResource"
+);
+
+@@OpenAPI.extension(LoadTests.createOrUpdate::parameters.resource,
```

Comment:
Maybe remove this to avoid lintDiff error on same name? You do need a client.tsp and use @clientName if different from wire name.

Reply:
Added clientName in client.tsp. The generated json although has wrong x-ms-client extension.